source/partials/ctas/_marketing-consent.haml

```diff
@@ -0,0 +1,10 @@
+.marketing-consent
+  %ul.signup-cta__incentives
+    %li.signup-cta__incentive
+      %input{ type: 'radio', name: 'marketing_consent', id: 'marketing_consent_yes', value: 'yes', checked: 'checked' }
+      %label{ for: 'marketing_consent_yes' }
```

Comment:
This partial show up multiple times on a single page (e.g. https://www.aptible-staging.com/enclave/), so you can't just use IDs like this (you need to have a unique ID every time the partial gets rendered — I suspect we can probably make this work in haml but I haven't looked into it yet, let me know if you'd like me to).

Indeed, if you go to that page (https://www.aptible-staging.com/enclave/), scroll to the bottom, and try to click one of the label, you'll note that:

- a) you scroll all the way up to the top of the page
- b) the thing you clicked didn't get checked